cmd/controller-manager/main.go

```diff
@@ -54,6 +55,72 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+// ensureCRDExists to ensure CRD exists before start
+func ensureCRDExists(ns string, kubeClient kubernetes.Interface, clientSet *versioned.Clientset) error {
+	condition := func() (bool, error) {
+		var err error
+		_, err = kubeClient.CoreV1().Namespaces().Get(ns, metav1.GetOptions{})
+		if err == nil {
+			// Only namespace "kube-system" exists, to list resources
```

Comment:
please fix this comment for `kube-system`

Reply:
ok